mitodl/micromasters
# Moved ipython to requirements.txt

#### What are the relevant tickets?
N/A

#### What's this PR do?
Moves ipython to requirements.txt
It also removes the test_requirements.in (no more pinned test requirements) and sets the pip cache environment variable to `/src/.cache`

#### How should this be manually tested?
1- the tests should pass on travis.
2- connect to the heroku build for this PR with the `./manage.py shell` and verify that ipython has been used (instead of the default python)
